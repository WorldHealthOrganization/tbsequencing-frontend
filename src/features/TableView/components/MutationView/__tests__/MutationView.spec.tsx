import React from 'react';
import {
  getTreeJSONWithRedux, render, screen, wrappedInRouter,
} from '../../../../../utils/testUtils/jestUtils';
import MutationView from '../MutationView';

describe('MutationView', () => {
  test('renders MutationView components without error', () => {
    render(
      wrappedInRouter(<MutationView />),
    );
  });

  test('renders MutationView title without error', () => {
    render(
      wrappedInRouter(<MutationView />),
    );

    screen.getByText('Mutations Search');
  });

  test('Should render correctly', () => {
    const tree = getTreeJSONWithRedux(
      wrappedInRouter(<MutationView />),
    );

    expect(tree).toMatchSnapshot();
  });
});
