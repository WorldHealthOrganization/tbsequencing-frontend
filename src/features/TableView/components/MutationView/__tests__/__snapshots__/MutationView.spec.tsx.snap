// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MutationView Should render correctly 1`] = `
<div
  className="css-usmhxf-MutationView"
>
  <p
    className="MuiTypography-root MuiTypography-body1 css-euu0lb-MuiTypography-root-BaseText"
  >
    Mutations Search
  </p>
  <div
    className="css-exdecl-MutationView"
  >
    <div>
      <div
        className="css-1dqm8h-AppPaper"
      >
        <div
          className="css-8gmxwn-MutationView"
        >
          <div
            className="css-tnkm0u-AppPaper"
          >
            <div
              className="css-1prolhb-MutationView"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-wb83mu-MuiTypography-root-BaseText"
              >
                Gene
              </p>
              <div
                aria-owns={null}
                className="MuiAutocomplete-root MuiAutocomplete-hasPopupIcon css-1vipv5o-MuiAutocomplete-root"
                data-testid="autocomplete"
                onClick={[Function]}
                onKeyDown={[Function]}
                onMouseDown={[Function]}
              >
                <div
                  className="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-pz9am2-MuiFormControl-root-MuiTextField-root"
                >
                  <div
                    className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-154xyx0-MuiInputBase-root-MuiOutlinedInput-root"
                    onClick={[Function]}
                  >
                    <input
                      aria-activedescendant={null}
                      aria-autocomplete="both"
                      aria-expanded={false}
                      aria-invalid={false}
                      autoCapitalize="none"
                      autoComplete="off"
                      autoFocus={false}
                      className="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
                      disabled={false}
                      id=":r0:"
                      onAnimationStart={[Function]}
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      onMouseDown={[Function]}
                      placeholder="Search by Gene"
                      readOnly={false}
                      required={false}
                      role="combobox"
                      spellCheck="false"
                      type="text"
                      value=""
                    />
                    <div
                      className="MuiAutocomplete-endAdornment css-1q60rmi-MuiAutocomplete-endAdornment"
                    >
                      <button
                        aria-label="Open"
                        className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-popupIndicator css-qzbt6i-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-popupIndicator"
                        disabled={false}
                        onBlur={[Function]}
                        onClick={[Function]}
                        onContextMenu={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={-1}
                        title="Open"
                        type="button"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-16563i6-MuiSvgIcon-root"
                          data-testid="SearchIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                          />
                        </svg>
                      </button>
                    </div>
                    <fieldset
                      aria-hidden={true}
                      className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        className="css-hdw1oc"
                      >
                        <span
                          className="notranslate"
                        >
                          ​
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
              </div>
            </div>
            <div
              className="css-1prolhb-MutationView"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-wb83mu-MuiTypography-root-BaseText"
              >
                Position
              </p>
              <div
                className="MuiFormControl-root MuiTextField-root css-1b58097-MuiFormControl-root-MuiTextField-root"
              >
                <div
                  className="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-adornedEnd css-o9k5xi-MuiInputBase-root-MuiOutlinedInput-root"
                  onClick={[Function]}
                >
                  <input
                    aria-invalid={false}
                    autoFocus={false}
                    className="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
                    disabled={false}
                    id=":r2:"
                    min={0}
                    onAnimationStart={[Function]}
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    placeholder="Search by Position"
                    required={false}
                    type="text"
                    value=""
                  />
                  <div
                    className="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
                  >
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1ghbz6l-MuiSvgIcon-root"
                      data-testid="SearchIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                      />
                    </svg>
                  </div>
                  <fieldset
                    aria-hidden={true}
                    className="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      className="css-hdw1oc"
                    >
                      <span
                        className="notranslate"
                      >
                        ​
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
            <div
              className="css-1rfwhsr-MutationView"
            >
              <p
                className="MuiTypography-root MuiTypography-body1 css-wb83mu-MuiTypography-root-BaseText"
              >
                Export CSV Data
              </p>
              <button
                className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-disableElevation MuiButtonBase-root css-sfprtt-MuiButtonBase-root-MuiButton-root-AppButton"
                disabled={false}
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
                type="button"
              >
                <span
                  className="MuiButton-startIcon MuiButton-iconSizeMedium css-1d6wzja-MuiButton-startIcon"
                >
                  <span
                    aria-hidden={true}
                    className="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium css-kp9ftd-MuiIcon-root"
                  >
                    file_download
                  </span>
                </span>
                <p
                  className="MuiTypography-root MuiTypography-body1 css-3cwtjx-MuiTypography-root-BaseText"
                >
                  Download
                </p>
              </button>
              <a
                download="MutationsSearch"
                href="data:text/csv;charset=utf-8,﻿"
                onClick={[Function]}
                target="_self"
              />
            </div>
          </div>
        </div>
        <div
          className="css-1ipk6i1-ExportButton"
        >
          <p
            className="MuiTypography-root MuiTypography-body1 css-5th3yp-MuiTypography-root-BaseText"
          >
            Export CSV Data
          </p>
          <div
            className="css-uhrw2b-ExportButton"
          >
            <button
              className="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-disableElevation MuiButtonBase-root css-sfprtt-MuiButtonBase-root-MuiButton-root-AppButton"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-startIcon MuiButton-iconSizeMedium css-1d6wzja-MuiButton-startIcon"
              >
                <span
                  aria-hidden={true}
                  className="material-icons notranslate MuiIcon-root MuiIcon-fontSizeMedium css-kp9ftd-MuiIcon-root"
                >
                  file_download
                </span>
              </span>
              <p
                className="MuiTypography-root MuiTypography-body1 css-3cwtjx-MuiTypography-root-BaseText"
              >
                Download
              </p>
            </button>
            <a
              download="generatedBy_react-csv.csv"
              href="data:text/csv;charset=utf-8,﻿\\"Gene\\",\\"Variant\\",\\"Nucleic Annotation\\",\\"Proteic Annotation\\",\\"Consequence\\",\\"Grade\\",\\"Global Frequency\\",\\"Total Samples\\",\\"Resistant Count\\",\\"Susceptible Count\\",\\"Intermediate Count\\""
              onClick={[Function]}
              target="_self"
            />
          </div>
        </div>
      </div>
      <div
        className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-11xur9t-MuiPaper-root-MuiTableContainer-root"
      >
        <div
          className="css-u0qmqf-DataGrid"
        >
          <table
            className="MuiTable-root css-adcf9a-MuiTable-root"
            role={null}
          >
            <thead
              className="MuiTableHead-root css-cwxlwz-MuiTableHead-root"
              role={null}
            >
              <tr
                className="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
                role={null}
              >
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="css-qcmszo-TableHeader"
                    >
                      Gene
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    >
                      <button
                        className="css-13hx71u-Thead"
                        data-testid="filter-button-geneName"
                        onClick={[Function]}
                        type="button"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1o8iwtf-MuiSvgIcon-root"
                          color="#009ADE"
                          data-testid="SearchIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                          />
                        </svg>
                      </button>
                    </div>
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="css-qcmszo-TableHeader"
                    >
                      Variant
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    >
                      <button
                        className="css-13hx71u-Thead"
                        data-testid="filter-button-variantName"
                        onClick={[Function]}
                        type="button"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1o8iwtf-MuiSvgIcon-root"
                          color="#009ADE"
                          data-testid="SearchIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                          />
                        </svg>
                      </button>
                    </div>
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="css-qcmszo-TableHeader"
                    >
                      Nucleic Annotation
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    >
                      <button
                        className="css-13hx71u-Thead"
                        data-testid="filter-button-nucleodicAnnName"
                        onClick={[Function]}
                        type="button"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1o8iwtf-MuiSvgIcon-root"
                          color="#009ADE"
                          data-testid="SearchIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                          />
                        </svg>
                      </button>
                    </div>
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="css-qcmszo-TableHeader"
                    >
                      Proteic Annotation
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    >
                      <button
                        className="css-13hx71u-Thead"
                        data-testid="filter-button-proteicAnnName"
                        onClick={[Function]}
                        type="button"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1o8iwtf-MuiSvgIcon-root"
                          color="#009ADE"
                          data-testid="SearchIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                          />
                        </svg>
                      </button>
                    </div>
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="css-qcmszo-TableHeader"
                    >
                      Consequence
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    >
                      <button
                        className="css-13hx71u-Thead"
                        data-testid="filter-button-consequence"
                        onClick={[Function]}
                        type="button"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1o8iwtf-MuiSvgIcon-root"
                          color="#009ADE"
                          data-testid="SearchIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                          />
                        </svg>
                      </button>
                    </div>
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="css-qcmszo-TableHeader"
                    >
                      Grade
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    >
                      <button
                        className="css-13hx71u-Thead"
                        data-testid="filter-button-variantGrade"
                        onClick={[Function]}
                        type="button"
                      >
                        <svg
                          aria-hidden={true}
                          className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1o8iwtf-MuiSvgIcon-root"
                          color="#009ADE"
                          data-testid="SearchIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                          />
                        </svg>
                      </button>
                    </div>
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="MuiButtonBase-root MuiTableSortLabel-root css-w9xm9n-MuiButtonBase-root-MuiTableSortLabel-root"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onContextMenu={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      Global Frequency
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                        data-testid="ArrowDownwardIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                        />
                      </svg>
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    />
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="MuiButtonBase-root MuiTableSortLabel-root css-w9xm9n-MuiButtonBase-root-MuiTableSortLabel-root"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onContextMenu={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      Total Samples
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                        data-testid="ArrowDownwardIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                        />
                      </svg>
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    />
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="MuiButtonBase-root MuiTableSortLabel-root css-w9xm9n-MuiButtonBase-root-MuiTableSortLabel-root"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onContextMenu={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      Resistant Count
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                        data-testid="ArrowDownwardIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                        />
                      </svg>
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    />
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="MuiButtonBase-root MuiTableSortLabel-root css-w9xm9n-MuiButtonBase-root-MuiTableSortLabel-root"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onContextMenu={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      Susceptible Count
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                        data-testid="ArrowDownwardIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                        />
                      </svg>
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    />
                  </div>
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-head MuiTableCell-alignLeft MuiTableCell-sizeMedium css-1fhv7sw-MuiTableCell-root"
                  scope="col"
                >
                  <div
                    className="css-1qyj6ee-Thead"
                  >
                    <span
                      className="MuiButtonBase-root MuiTableSortLabel-root css-w9xm9n-MuiButtonBase-root-MuiTableSortLabel-root"
                      onBlur={[Function]}
                      onClick={[Function]}
                      onContextMenu={[Function]}
                      onDragLeave={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      onKeyUp={[Function]}
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchMove={[Function]}
                      onTouchStart={[Function]}
                      role="button"
                      tabIndex={0}
                    >
                      Intermediate Count
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTableSortLabel-icon MuiTableSortLabel-iconDirectionAsc css-1vweko9-MuiSvgIcon-root-MuiTableSortLabel-icon"
                        data-testid="ArrowDownwardIcon"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20 12l-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
                        />
                      </svg>
                    </span>
                    <div
                      className="css-trc57z-Thead"
                    />
                  </div>
                </th>
              </tr>
            </thead>
            <tbody
              className="MuiTableBody-root css-apqrd9-MuiTableBody-root"
              role={null}
            >
              <tr
                className="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                role={null}
              >
                <td
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-4b1vly-MuiTableCell-root"
                  colSpan={100}
                  rowSpan={100}
                >
                  No results matching your request
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
    <div
      className="css-opkoxw-MutationView"
    >
      <nav
        aria-label="pagination navigation"
        className="MuiPagination-root MuiPagination-outlined css-10s0nwi-MuiPagination-root"
      >
        <ul
          className="MuiPagination-ul css-wjh20t-MuiPagination-ul"
        >
          <li>
            <button
              aria-label="Go to previous page"
              className="MuiButtonBase-root Mui-disabled MuiPaginationItem-root MuiPaginationItem-sizeMedium MuiPaginationItem-outlined MuiPaginationItem-rounded Mui-disabled MuiPaginationItem-previousNext css-19xm0h7-MuiButtonBase-root-MuiPaginationItem-root"
              disabled={true}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={-1}
              type="button"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiPaginationItem-icon css-g2z002-MuiSvgIcon-root-MuiPaginationItem-icon"
                data-testid="NavigateBeforeIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                />
              </svg>
            </button>
          </li>
          <li>
            <button
              aria-current="true"
              aria-label="page 1"
              className="MuiButtonBase-root MuiPaginationItem-root MuiPaginationItem-sizeMedium MuiPaginationItem-outlined MuiPaginationItem-rounded Mui-selected MuiPaginationItem-page css-19xm0h7-MuiButtonBase-root-MuiPaginationItem-root"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              1
            </button>
          </li>
          <li>
            <button
              aria-label="Go to next page"
              className="MuiButtonBase-root Mui-disabled MuiPaginationItem-root MuiPaginationItem-sizeMedium MuiPaginationItem-outlined MuiPaginationItem-rounded Mui-disabled MuiPaginationItem-previousNext css-19xm0h7-MuiButtonBase-root-MuiPaginationItem-root"
              disabled={true}
              onBlur={[Function]}
              onClick={[Function]}
              onContextMenu={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={-1}
              type="button"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiPaginationItem-icon css-g2z002-MuiSvgIcon-root-MuiPaginationItem-icon"
                data-testid="NavigateNextIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M10 6L8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                />
              </svg>
            </button>
          </li>
        </ul>
      </nav>
    </div>
  </div>
</div>
`;
